hazelcast/src/main/java/com/hazelcast/cache/impl/operation/CacheBackupEntryProcessorOperation.java

```diff
@@ -64,6 +66,21 @@ public void run()
     }
 
     @Override
+    public void afterRun() throws Exception {
+        if (cacheService.wanReplicationEnabled(name)) {
+            CacheRecord record = cache.getRecord(key);
+            if (record != null) {
+                DefaultCacheEntryView entryView = new DefaultCacheEntryView(key,
```

Comment:
I'll create `CacheEntryViews` class which will provide static factory methods to create `CacheEntryView` instances.